articles/logic-apps/set-up-model-context-protocol-server-standard.md

```diff
@@ -23,6 +23,9 @@ ms.update-cycle: 180-days
 
 Typically, large language models (LLMs) work with AI agents that handle and fulfill requests by using prebuilt *tools* that agents call to complete tasks, like send an email, query a database, or trigger a workflow. In Azure Logic Apps, you can jumpstart building these tools by reconfiguring a Standard logic app as your own *remote* Model Context Protocol (MCP) server. This capability means that you can expose existing workflows as tools that LLMs, AI agents, and MCP clients can use to interact with enterprise resources and assets. In this context, *remote* means that the MCP server runs outside the environment where the interface for your AI agent interface.
 
+You can now configure multiple MCP servers within a single Standard Logic App.  
```

Comment:
This info appears too early in the introduction because the reader is just learning about MCP servers and disrupts the reading flow with "feature" info. Also, related sentences should appear on the same numbered "line", not separate lines.

You changed the "single MCP" scenario to a "multi-MCP" scenario. What if the customer just wants one MCP? Do they follow the same directions but define only one MCP server? Or, do they follow different directions for a single MCP server? If the steps are different, you need to have both sets of steps.